src/components/Heading/Heading.scss

```diff
@@ -7,11 +7,12 @@
     }
     color: var(--color-foreground);
     text-align: left;
+    margin: var(--heading-margin) 0;
```

Comment:
I'm not sure if we should make this themable just yet, since it would affect **all** headings in the application. I think we should better set it to `0` without using variables for now.